db/migrations/migrations/2023.06.21T15.28.44.urltable.js

```diff
@@ -1,7 +1,7 @@
 /** @type {import('umzug').MigrationFn<any>} */
 exports.up = async ({context: sequelize}) => {await sequelize.query(`CREATE TABLE IF NOT EXISTS url (
     paste_id varchar(60) PRIMARY KEY,
-    hash VARCHAR ( 50 ),
+    url_id VARCHAR ( 60 ),
```

Comment:
Last time you edit "past" migrations =)
Add new ones when changes in DB are needed